src/vcpkg/postbuildlint.cpp

```diff
@@ -240,6 +240,27 @@ namespace vcpkg::PostBuildLint
         return LintStatus::SUCCESS;
     }
 
+    static LintStatus check_for_usage_forgot_install(const Filesystem& fs,
+                                                     const Path& port_dir,
+                                                     const Path& package_dir,
+                                                     const PackageSpec& spec)
+    {
+        static const std::string STANDARD_INSTALL_USAGE =
+            R"###(file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"))###";
+
+        auto usage_path_from = port_dir / "usage";
+        auto usage_path_to = package_dir / "share" / spec.name() / "usage";
+
+        if (fs.exists(usage_path_from, IgnoreErrors{}) && !fs.exists(usage_path_to, IgnoreErrors{}))
```

Comment:
```suggestion
        if (fs.is_regular_file(usage_path_from) && !fs.is_regular_file(usage_path_to))
```
Please check if the file is a regular file. We had issues with copyright files where people abused this to create copyright folders.
Not sure if `is_regular_file` is the correct name but we have a function for this use case that does what the name suggests.